site/zh/blog/new-in-devtools-93/index.md

```diff
@@ -0,0 +1,148 @@
+
+
+## 网络面板可预览 Web bundle {: #web-bundle }
+[Web bundle](https://web.dev/web-bundles/) 是一种新兴的文件格式，用于封装一个或多个 HTTP 资源。现在，您可以在**网络**面板那里预览 web bundle的网络请求。
+
+web bundle 特性现还处于实验性质。测试该特性之前，请在 `chrome://flags` 那里打开 `#enable-experimental-web-platform-features` 开关。
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/PEv1mNA14K18t5P3N6Yj.png", alt="web bundle preview", width="800", height="492" %}
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/e7672c40f2febc80786632c188b6029b2f2ac7b7 #}
+
+Chromium 议题: [1182537](https://crbug.com/1182537)
+
+
+## Attribution Reporting API 调试 {: #attribution-reporting }
+现在，Attribution Reporting API 错误会显示在**问题**面板那里。
+
+[Attribution Reporting](https://developer.chrome.com/docs/privacy-sandbox/attribution-reporting/)是一个新的 API，旨在帮助您在没有使用跨站标识符的情况下，评估用户行为（比如点击或者查看广告）所导致的转化率。
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/bkEGVEv5kKc9M6qBUmLz.png", alt="Attribution Reporting API errors in the Issues tab", width="800", height="501" %}
+
+Chromium 议题: [1190735](https://crbug.com/1190735)
+
+
+## 更好地处理控制台里面的字符串 {: #string }
+**控制台**里面的新右键菜单允许您复制任何字符串，作为文本内容、JavaScript 字面量或者 JSON 字面量。
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/O5uMSgkHrQ2mQDSjmg3A.png", alt="New context menu in the Console", width="800", height="477" %}
+
+在 Chrome 90，**控制台**默认将[输出字符串格式化为 JSON 字面量](/blog/new-in-devtools-90/#double-quotes)。我们从开发者那里得到反馈：这个改动让人有点迷，一些人觉得字符串的转义有点多，不利于字符串的可读性。
+
+现在，**控制台**默认将输出字符串格式化为有效的 JavaScript 字面量，并且会进一步提供 3 种复制字符串的选项。**复制字符串作为 JavaScript 字面量**的这个选项，会转义那些合适且特殊的字符，然后会根据字符串内容来决定是否使用单引号、双引号或者反引号来包裹字符串。**复制字符串内容**的这个选项呢，会一字不差地将原生的字符串内容（包括换行符以及其它的特殊字符）复制到剪切板。最后，**复制字符串作为 JSON 字面量**的这个选项，会将字符串格式化为有效的 JSON 字面量，然后复制到剪切板。
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/9242d13569e9fe67ac01e75d28fa2b6e6bf310d2 #}
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/5715a7b9800532d8b28e2c9fa2d3c1e220ba54a8 #}
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/29236e333a856ae5a952fe4182545b1e2bde5539 #}
+
+Chromium 议题: [1208389](https://crbug.com/1208389)
+
+
+## 改善 CORS 调试体验 {: #cors }
+**控制台**里面的 CORS 相关错误，现在可以链接到**网络** 和**问题**面板。
+
+点击 CORS 相关错误信息旁边的两个新按钮以查看网络请求，以及进一步理清楚该错误信息，在**问题**面板那里获取潜在的解决方案。
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/VzoUggSoM0FnkDlIFPhq.png", alt="Icons next to the CORS-related error message", width="800", height="485" %}
+
+Chromium 议题: [1213393](https://crbug.com/1213393)
+
+
+## Lighthouse 8.1 {: #lighthouse }
+现在，**Lighthouse** 面板运行的是 Lighthouse 8.1。
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/wENi9RXYMxdhm3zI4NVu.png", alt="Lighthouse", width="800", height="628" %}
+
+如果您的网站有使用 source maps，在 **Lighthouse** 报告里将显示 **View Treemap** 按钮。点击该按钮查看您网页所有的 JavaScript 的拆解情况，检测 JavaScript 的体积以及载入时 JavaScript 的覆盖情况。
+
+这个报告也引入一个新的指标过滤条件（参考上述截图里面的 **Show audits relevant to** 过滤条件）。选择一个指标，以便于获取信息以及相关诊断，从而达到改善该指标的目的。
+
+针对评分那块，**Performance Category** 做了很多调整，方便与其它的性能工具保持一致以及更好地反映出 web 的情况。
```

Comment:
"评分那块" seems a bit informal here. Could we change it to "评分（的）部分"?

Reply:
nit: I feel like the original text, "state of the web", implies currently common/widely accepted standards of the web, as in "state of the art". So maybe we could translate it as "web 的现状" instead of "web 的情况"? But of course this is only my interpretion of the original text, so CC @chybie here for clarification and suggestion.